### DISCUSSION TITLE
Hide taskbar on secondary monitors only?

### DISCUSSION THREAD

**Discussion by Muhlawor** [valinet/ExplorerPatcher#2183] [Q&A]
Is it possible to hide the taskbar on secondary monitors only, and keep it on the primary?

**Comment by UhuruNUru** [comment]
That's standard behavior in W11 Taskbar, but it's exactly the behaviour I don't want.
If it's actually what you want, you don't even need Explorer Patcher to do that.
**Settings > Personalisation > Taskbar > Taskbar Behaviours > Untick Show My taskbar on all displays.**

The "Primary" monitor is where my games will launch, when user has no choice.
That's exactly the wrong place, plus I can't stand W11 Taskbar, or W8+ Start Menu.

I need to have my primary monitor central on my FreeSync/GSync Monitor, and to only have the Taskbar on my Left Screen, requires the W10 Taskbar.
Then you can move it where you want, including, on top, and either side.
I used **Open-Shell**, and **7+Taskbar Tweaker** to achieve my super minimal W7 Start Menu with W10 Taskbar.
Here's an Imgur Link, showing my setup, The Mouse Over Roll Down Icons requires **Fences v4.0**, other tools are free, and not everyone has 600+ Modded Games installed, and needs as many Desktop Shortcuts as I use, with Extra large Icons, but If I go to the trouble of making my own custom Icons, I want to see them.

So either way, you're covered, and you can see how I do it below.
[W11 Pro - The Uhuru N’Uru Way](https://imgur.com/a/7XI9MA9)
